templates/partials/categories/item.tpl

```diff
@@ -9,9 +9,7 @@
 		<h2 class="title">
 			<!-- IMPORT partials/categories/link.tpl --><br/>
 			<!-- IF ../descriptionParsed -->
-			<span class="description">
-			{../descriptionParsed}
-			</span>
+			<span class="description">{../descriptionParsed}</span>
```

Comment:
Well, not exactly... it was fine with the line breaks, but what I mean is a `div` is a block element (multiple divs stack on top of one another) while a span is inline (multiple spans are next to each other), so changing a div to a span isn't enough, you need to also apply the bootstrap class to make it a block element.

Reply:
@julianlam Sorry, my bad, was updated too, looks exactly the same now